llvm/lib/Target/PowerPC/AsmParser/PPCAsmParser.cpp

```diff
@@ -820,6 +820,27 @@ void PPCAsmParser::ProcessInstruction(MCInst &Inst,
                                       const OperandVector &Operands) {
   int Opcode = Inst.getOpcode();
   switch (Opcode) {
+  case PPC::PLA8:
+  case PPC::PLA: {
+    MCInst TmpInst;
+    TmpInst.setOpcode(PPC::PADDI);
```

Comment:
Is this line needed if we are going to end up setting the opcode below on 827 anyway?